content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Go"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+{{ blockEnd }}
+
+{{ startTab "Javascript/Typescript"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Python"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Rust"}}
+
+**SDK: Outbound HTTP**
```

Comment:
Suggest dropping the `SDK` prefix.  (It's all SDK.)

Reply:
I have dropped the `SDK:` prefixes in each spot. Shall we leave the `Trigger:` prefix on the other ones? Which is currently the case.

Reply:
I think it would read better as a "Triggers" subhead (but then we'd need a corresponding subhead for the APIs), or as "HTTP trigger".